### PR TITLE
Sidebar mainpage

### PR DESCRIPTION
**Pull request by MiheerTamkhane**
- Sidebar for notes-app

### CODE REVIEW

File: src/frontend/components/Sidebar/Sidebar.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  padding: 0.5rem;
+  border: 1px solid var(--gray-500);
+  border-radius: 50%;
+  cursor: pointer;
+  transition: all 0.3s ease;
+  background-color: var(--gray-100);
+}
+/* .arrow:hover {
+  padding-left: 1rem;
+} */
```
Comment (prathmeshjagtap) [inline src/frontend/components/Sidebar/Sidebar.css]: you can remove commented  code

File: src/index.css [hunk 2]
```diff
@@ -46,6 +46,7 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  /* background-color: var(--color-dark); */
```
Comment (prathmeshjagtap) [inline src/index.css:49]: you can remove this

**Review by prathmeshjagtap — APPROVED** [review]
The code is well structured Liked your sidebar its very good, just remove the commented  code if you are not going  to use it in future

**Review by horsemaker — APPROVED** [review]
The sidebar looks awesome! 👌
The code also looks clean.

**Review by manojasarsa — APPROVED** [review]
Code looks clean and sidebar is awesome!

**Review by Shrinkhlarajpoot — COMMENTED** [review]
looks good

**Pull request merged**